app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -126,9 +126,9 @@ export const ContentForm: React.FC<IContentFormProps> = ({ contentType = Content
       // Now upload the file if it exists.
       if (!!values.file) {
         const uploadResult = await upload(contentResult, values.file);
-        setContent(toForm(uploadResult));
+        setContent(toForm({ ...uploadResult, tonePools: values.tonePools }));
```

Comment:
I assume there is an issue with arrays or something here.  Can you explain?

Reply:
For some reason when no tone values were assigned to an existing piece of content the array was having issues being assigned the new value into state. Didn't spend too much time on alternative methods to solve this; could look into more if need be